examples/skysync-cli-example-create-job-01.ps1

```diff
@@ -0,0 +1,77 @@
+<#
```

Comment:
Should these scripts be grouped in a `powershell` directory?

Reply:
dir added